src/tools/mmaps_generator/PathGenerator.cpp

```diff
@@ -73,7 +73,7 @@ bool handleArgs(int argc, char** argv,
                bool &bigBaseUnit,
                char* &offMeshInputPath,
                char* &file,
-               int& threads)
+               unsigned int& threads)
```

Comment:
Is there any reason to pass this by reference? The amount of threads doesn't change during program execution neither is adjusting of the threadpool supported.